dist/elastio-snap.spec

```diff
@@ -6,6 +6,9 @@
 # Location to install kernel module sources
 %global _kmod_src_root %{_usrsrc}/%{name}-%{version}
 
+# Location for systemd shutdown script
+%global _systemd_shutdown /lib/systemd/system-shutdown
```

Comment:
Does it always have the same location on all distro versions we do [support](https://github.com/elastio/elastio-snap/blob/master/INSTALL.md#from-repositories)?

Reply:
@e-kov Looks like yes: https://en.wikipedia.org/wiki/Systemd

Reply:
We are not going to add systemV scripts, but for your information.
In theory the driver claims support starting from kernel 2.6.18 https://github.com/elastio/elastio-snap/blob/7fe09e0082f13be0afe9789bd033e39151590ae8/src/elastio-snap.c#L12 and... systemV as well like in CentOS 6.
This year I was asked to fix a compilation of the driver for CentOS 6 https://github.com/elastio/elastio-snap/issues/116

Probably it's time to drop 2nd kernels support and raise the minimal one up to `3.10.0` as in CentOS 7. Even Debian 8 has  `3.16.0`. @skypodolsky what do you think?

Reply:
A little bit late with the reply, but I agree. I have already changed the relevant comments in INSTALL.md and elastio-snap.c, so from now on the lowest official version is v3.10.